powertools-validation/src/main/java/software/amazon/lambda/powertools/validation/internal/ValidationAspect.java

```diff
@@ -91,8 +91,14 @@ public Object around(ProceedingJoinPoint pjp,
                     event.getRecords().forEach(record -> validate(decode(record.getData()), inboundJsonSchema));
                 } else if (obj instanceof KafkaEvent) {
                     KafkaEvent event = (KafkaEvent) obj;
-                    event.getRecords().forEach((s, records) -> records.forEach(record -> validate(record.getValue(), inboundJsonSchema)));
-                }else if (obj instanceof KinesisAnalyticsFirehoseInputPreprocessingEvent) {
+                    event.getRecords().forEach((s, records) -> records.forEach(record -> validate(decode(record.getValue()), inboundJsonSchema)));
+                } else if (obj instanceof ActiveMQEvent) {
```

Comment:
I am correct in understanding that with this change, we are sort of covering this feature request too ? https://github.com/awslabs/aws-lambda-powertools-java/issues/179

Reply:
Yes, ActiveMQ and RabbitMQ have been added to the validation.